package.json

```diff
@@ -13,19 +13,19 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/AlaskaAirlines/WC-Generator.git"
+    "url": "https://github.com/AlaskaAirlines/WC-Generator.git"
   },
   "author": "AlaskaAirlines",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=16.13.0"
+    "node": ">=18.15.0"
   },
   "bugs": {
     "url": "https://github.com/AlaskaAirlines/WC-Generator/issues"
   },
   "homepage": "https://auro.alaskaair.com/getting-started/developers/generator/install",
   "dependencies": {
-    "arg": "^4.1.3",
+    "arg": "^5.0.2",
     "chalk": "^3.0.0",
```

Comment:
Why is this a direct dependency rather than a dev dependency?

Reply:
It's needed on the client who is using the generator, not just for a build.